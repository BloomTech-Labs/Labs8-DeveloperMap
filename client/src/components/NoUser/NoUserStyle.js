import styled from 'styled-components';

export const Header = styled.h1`
  padding: 50px 20%;
  font-size: 3rem;
`;
