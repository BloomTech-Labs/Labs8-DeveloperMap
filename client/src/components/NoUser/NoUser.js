import React from 'react';
import { ModalContainer, ModalMain } from '../../styles/ModalGlobalStyle';
import { Header } from './NoUserStyle.js';

const NoUser = () => {
  return (
    <ModalContainer data-type="modal-container">
      <ModalMain>
        <Header>This user is no longer with us.</Header>
      </ModalMain>
    </ModalContainer>
  );
};

export default NoUser;
