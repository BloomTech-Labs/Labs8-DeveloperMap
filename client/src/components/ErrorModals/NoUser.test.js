import React from 'react';
import renderer from 'react-test-renderer';
import NoUser from './NoUser';

describe(`no user Component`, () => {
  it('renders correctly', () => {
    const tree = renderer.create(<NoUser />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
