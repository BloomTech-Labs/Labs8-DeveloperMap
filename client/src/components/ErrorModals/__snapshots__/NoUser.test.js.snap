// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`no user Component renders correctly 1`] = `
<div
  className="sc-bdVaJa cbxIaT"
  data-type="modal-container"
>
  <div
    className="modal sc-bwzfXH gLXQyA"
  >
    <h1
      className="sc-htpNat bBFOFo"
    >
      This user is no longer with us.
    </h1>
  </div>
</div>
`;
